skins/laika/src/components/pages/donation_form/subpages/PaymentPage.vue

```diff
@@ -1,5 +1,10 @@
 <template>
-	<div class="payment-page">
+	<form name="laika-donation-payment-data"
+		id="laika-donation-payment-data"
+		class="payment-page column is-full"
```

Comment:
I'd lose the "data" suffix here, because it's kind of implied. "Personal data" is a compund word, but "payment" would suffice imho. But that's just my personal taste and me trying to avoid any hints of AbstractDataFactoryModelManagerContainerHelper.